pycipapi/models.py

```diff
@@ -141,6 +141,17 @@ def __init__(self, **kwargs):
         self.cva_transaction_id = kwargs.get('cva_transaction_id')
 
 
+class ParticipantConsent(object):
+    def __init__(self, **kwargs):
+        self.created_at = kwargs.get('created_at')
```

Comment:
these two fields are still not mandatory in the payload, right?

Reply:
Yes you don't need to include it in the payload it's auto- populated by cipapi.